experiments/rsc/src/worker.tsx

```diff
@@ -8,11 +8,25 @@ import { renderToRscStream } from "./render/renderToRscStream";
 import { TwilioClient, quickReplyMessage, saveVCardToR2 } from "./twilio";
 import { ssrWebpackRequire } from "./imports/worker";
 import { rscActionHandler } from "./register/worker";
-
+import WelcomePage from "./app/WelcomePage";
+import TradesmenPage from "./app/TradesmenPage";
+import ProfessionsPage from "./app/ProfessionsPage";
+import AddTradesmanPage from "./app/AddTradesmanPage";
+import TradesmanPage from "./app/TradesmanPage";
+import { setupR2Storage } from "./r2storage";
 // todo(peterp, 2024-11-25): Make these lazy.
 const routes = {
   "/": HomePage,
+  "/welcome": WelcomePage,
   "/admin": AdminPage,
+  "/tradesmen/:profession": (props: { params: { profession: string } }) => (
+    <TradesmenPage profession={props.params.profession} />
+  ),
+  "/tradesman/:id": (props: { params: { id: number } }) => (
+    <TradesmanPage id={props.params.id} />
+  ),
+  "/professions": ProfessionsPage,
+  "/add-tradesman": AddTradesmanPage,
```

Comment:
Nice API!

Note for myself that I need to investigate if there's a way we can lazy-load modules in cloudflare, we'll probably need this for routing.